packages/plugin-build-types/src/index.ts

```diff
@@ -41,6 +58,8 @@ export async function build({
           "--declarationMap",
           "false",
           "--declarationDir",
+          "--project",
```

Comment:
Hey @umidbekkarimov and @FredKSchott, Getting following error because of this change (in latest ```@pika/plugin-build-types``` version i.e. ```0.3.15```):

```
error TS6053: File 'dist-types/.ts' not found.

error TS6054: File 'tsconfig.json' has unsupported extension. The only supported extensions are '.ts', '.tsx', '.d.ts'.
```

I think fix should be like this: 

```
      await execa(
        tscBin,
        [
          "-d",
          "--emitDeclarationOnly",
          "--declarationMap",
          "false",
          "--project",
          tsConfigPath,
          "--declarationDir", // --declarationDir should come after tsConfigPath
          path.join(out, "dist-types/")
        ],
        { cwd }
      );
```